app/src/main/java/com/sun/premierleague/ui/stats/StatsAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package com.sun.premierleague.ui.stats
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sun.premierleague.R
+import com.sun.premierleague.base.BaseAdapter
+import com.sun.premierleague.base.BaseViewHolder
+import com.sun.premierleague.data.model.StatItem
+import com.sun.premierleague.utils.loadImage
+import kotlinx.android.synthetic.main.item_recyclerview_stats.view.*
+
+class StatsAdapter : BaseAdapter<StatItem>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BaseViewHolder<StatItem> {
+        return StatsViewHolder(
+            LayoutInflater.from(parent.context)
+                .inflate(R.layout.item_recyclerview_stats, parent, false)
+        )
+    }
```

Comment:
Viết cho đoạn code dễ hiểu nhất có thể thay vì tiện nhất có thể nhé
```suggestion
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BaseViewHolder<StatItem> {
        val view = LayoutInflater.from(parent.context).inflate(R.layout.item_recyclerview_stats, parent, false)
        return StatsViewHolder(
            
        )
    }
```